### PR TITLE
Update configure index test and clean up control plane client

### PR DESCRIPTION
**Pull request by rohanshah18**
## Problem

Test for configure index were missing. Also the open api spec was updated.

## Solution

Add tests for configure index and updated open api generated code.

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Infrastructure change (CI configs, etc)
- [X] Non-code change (docs, etc)
- [ ] None of the above: (explain here)

## Test Plan

Ran integration tests.

### CODE REVIEW

File: src/main/java/io/pinecone/exceptions/HttpErrorMapper.java [hunk 1]
```diff
@@ -9,6 +9,8 @@ public static void mapHttpStatusError(FailedRequestInfo failedRequestInfo) {
                 throw new PineconeBadRequestException(failedRequestInfo.getMessage());
             case 401:
                 throw new PineconeAuthorizationException(failedRequestInfo.getMessage());
+            case 403:
```
Comment (rohanshah18) [inline src/main/java/io/pinecone/exceptions/HttpErrorMapper.java:12]: Added Forbidden exception

File: src/main/java/io/pinecone/PineconeControlPlaneClient.java [hunk 2]
```diff
@@ -11,22 +11,14 @@
 import org.openapitools.client.model.IndexList;
 import org.openapitools.client.model.IndexModel;
 
-public class PineconeIndexOperationClient {
+public class PineconeControlPlaneClient {
```
Comment (rohanshah18) [inline src/main/java/io/pinecone/PineconeControlPlaneClient.java:14]: Renamed to better reflect collections and index operations

File: src/integration/java/io/pinecone/integration/controlPlane/index/serverless/CreateDescribeListAndDeleteIndexTest.java [hunk 3]
```diff
@@ -13,11 +13,11 @@
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 public class CreateDescribeListAndDeleteIndexTest {
-    private PineconeIndexOperationClient controlPlaneClient;
+    private PineconeControlPlaneClient controlPlaneClient;
```
Comment (rohanshah18) [inline src/integration/java/io/pinecone/integration/controlPlane/index/serverless/CreateDescribeListAndDeleteIndexTest.java:16]: will add validation and the corresponding tests in future PR's

**Review by austin-denoble — APPROVED** [review]
Looks good, thank for continuing to clean things up and re-enabling the configure index tests. 👍

**Pull request merged**